src/main/groovy/com/github/jrubygradle/jar/JRubyJarConfigurator.groovy

```diff
@@ -159,6 +159,18 @@ class JRubyJarConfigurator {
         mainClass(EXTRACTING_MAIN_CLASS)
     }
 
+    void jrubyVersion(String version) {
```

Comment:
Once you have moved the code, this should be done differently. They can just be properties on `JRubyJar`. Then in the `JRubyJarPlugin` you use `afterEvaluate` to add the correct dependencies. See `JRubyExec.updateJRubyDependencies()` in the `base` plugin as an example.